Cargo.toml

```diff
@@ -168,8 +168,8 @@ unicode-segmentation = "1.12.0"
 url = "2.5"
 urlpattern = "0.3"
 uuid = { version = "1.12.1", features = ["v4"] }
-vello = { git = "https://github.com/linebender/vello", rev = "daf940230a24cbb123a458b6de95721af47aef98" }
-vello_cpu = { git = "https://github.com/linebender/vello", rev = "daf940230a24cbb123a458b6de95721af47aef98" }
+vello = { git = "https://github.com/linebender/vello", rev = "65975d7612e38c199afaa547268763e1b4b826c9" }
+vello_cpu = { git = "https://github.com/linebender/vello", rev = "65975d7612e38c199afaa547268763e1b4b826c9" }
```

Comment:
We should not be using git commits for these upstream dependencies. I think it makes sense to wait for releases.